targets/TARGET_Cypress/scripts/PSOC6.py

```diff
@@ -306,13 +313,29 @@ def complete(message_func, elf0, hexf0, hexf1=None):
 
 def merge_action(args):
     """Entry point for the "merge" CLI command."""
+    try:
+        elf_file = list(pathlib.Path(args.artefacts_location).glob("*.elf"))[0]
```

Comment:
Can't we just pass the absolute paths to the ELF and hex files as command line arguments? Not sure why we need to do this globbing here and in the other script.

Reply:
That would be ideal. Any suggestions? I could not find anything but I will keep digging the docs.

Reply:
I couldn't find any CMake native way to do this either unfortunately. We might have to live with the globbing for now. We can always improve this later.